lib/data/schema.js

```diff
@@ -38,6 +38,9 @@ const { stripNamespacesFromPath, traverseXml, findOne, findAll, and, or, root, n
 // Turns relpaths into abspaths; leaves abspaths alone.
 const absolutify = (path, rootName) => ((path.charAt(0) === '/') ? path : `/${rootName}/${path}`);
 
+// Look for entity saveto attribute in a binding
+const saveToAttr = (binding) => Object.keys(binding).find((k) => stripNamespacesFromPath(k) === 'saveto');
```

Comment:
Double-check `saveto` vs. other ways of writing it (`save_to`, `saveTo`)

Reply:
This [random webpage](https://docs.seneca.nl/Formulierenserver-Docs/XForms/XForms-Reference/The-bind-element.html) has some attributes in xform bind tags like `readonly` and `nodeset` (maybe that is just one word) that are all lowercase so we will stick with `saveto`.